### ISSUE TITLE
improve at-rule-conditional-no-parentheses

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#378]
Will be great if we implement new rule with name `at-rule-conditional-parentheses` with options: `never` and `always`. We try to always use parens to avoid misleading in code and keep our code based consistently with other programmers language where parens are required

**Comment by nex3** [comment]
FWIW, we're working on an official Sass style guide endorsed by the team, and it likely won't allow parentheses in top-level control flow statements.

**Comment by evilebottnawi** [comment]
@nex3 thanks for feedback, can you clarify why?

**Comment by nex3** [comment]
The same reason we don't encourage parentheses for, say, `@return`: they're syntactically unnecessary, and unnecessary punctuation generally adds clutter without conveying information. This is especially true for parentheses in Sass which are used to indicate maps and often lists. Also, in practice, it's most common for authors to omit these parentheses.

**Comment by kristerkari** [comment]
> Will be great if we implement new rule with name `at-rule-conditional-parentheses` with options: `never` and `always`

I was also thinking about that before merging `at-rule-conditional-no-parentheses`, but I decided that the `no-` rule is probably enough since the Sass team members think that it's better to not write the parens.

**Comment by evilebottnawi** [comment]
Yep, but `sass` team has own opinion other can have other :smile:

**Comment by kristerkari** [comment]
If someone strongly feels like such rule is needed, we can add it. I don't use the stylistic rules myself because I let prettier do the code formatting.

**Comment by evilebottnawi** [comment]
@kristerkari now prettier print parens if they exists and do not print if they not, so if still need this rule

**Comment by kristerkari** [comment]
Let's add the rule that you suggested if there are enough people who actually need it and can give some examples why they need it.

Let's not add rules just because someone _might_ need them.

**Comment by evilebottnawi** [comment]
:+1:
